### PR TITLE
refactors the addClickListener

### PR DESCRIPTION
**Pull request by radishmouse**
Changes name of function, factors out DOM selectors.

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -46,48 +46,17 @@ function openOverlayWithImage(imgUrl) {
   overlayElement.classList.remove('hidden');
 }
 
-function addThumbnailClickListeners() {
+function addClickListeners(container, filter) {
 
-  // $(document.body).on('click', function (event) {
-  //   console.log('at the body');
-  // });
```
Comment (nybblr) [inline scripts/main.js]: Yay nuking spurious code! 🎊

File: scripts/main.js [hunk 2]
```diff
@@ -46,48 +46,17 @@ function openOverlayWithImage(imgUrl) {
...
-  // thumbnailItems.forEach(function (thumb) {
-  //   var url = thumb.getAttribute('href');
-  //   thumb.addEventListener('click', function (event) {
-  //     event.preventDefault();
-  //     openOverlayWithImage(url);
-  //   })
-  //   console.log('done with addEventListener');
-  // })
-  // console.log('done with forEach');
-  // console.log('done with addThumbnailClickListeners');
```
Comment (nybblr) [inline scripts/main.js:90]: Would it be easier to read if we split up the old code removal into a separate commit, with an explanation of why?

File: scripts/main.js [hunk 3]
```diff
@@ -46,10 +46,10 @@ function openOverlayWithImage(imgUrl) {
   overlayElement.classList.remove('hidden');
 }
 
-function addThumbnailClickListeners() {
+function addClickListeners(container, filter) {
```
Comment (radishmouse) [inline scripts/main.js]: Perhaps, consider using a more specific function name. Such as...addListenersToElements.

Comment (nybblr) [reply scripts/main.js]: I like it@

File: scripts/main.js (hunk 2 above)
Comment (radishmouse) [reply scripts/main.js:90]: Yes.

File: scripts/main.js (hunk 3 above)
Comment (radishmouse) [reply scripts/main.js]: I think I'll go with `addListeners`. More generic, because that makes sense.

**Comment by nybblr** [comment]
:+1: Merge the thing!

File: scripts/main.js [hunk 4]
```diff
@@ -46,7 +46,7 @@ function openOverlayWithImage(imgUrl) {
   overlayElement.classList.remove('hidden');
 }
 
-function addClickListeners(container, filter) {
+function addListeners(container, filter) {
```
Comment (wegmans-andrew-keller) [inline scripts/main.js]: 🦎

**Pull request merged**